ganttproject/src/net/sourceforge/ganttproject/search/TaskSearchService.java

```diff
@@ -24,13 +24,14 @@
 
 import net.sourceforge.ganttproject.IGanttProject;
 import net.sourceforge.ganttproject.gui.UIFacade;
+import net.sourceforge.ganttproject.language.GanttLanguage;
 import net.sourceforge.ganttproject.task.Task;
 
 /** Search service for tasks */
 public class TaskSearchService extends SearchServiceBase<TaskSearchService.MySearchResult, Task> {
   static class MySearchResult extends SearchResult<Task> {
-    public MySearchResult(Task t, TaskSearchService searchService, String query, String snippet, String snippetText) {
-      super(t.getTaskID(), "Task", t.getName(), query, snippet, snippetText, "", t, searchService);
+    MySearchResult(Task t, TaskSearchService searchService, String query, String snippet, String snippetText) {
+      super(t.getTaskID(), GanttLanguage.getInstance().getCorrectedLabel("task"), t.getName(), query, snippet, snippetText, t, searchService);
```

Comment:
Apparently I was wrong about existing keys for resources and tasks, sorry. They do exist in plural form but not in singular. So let's create new keys: `generic.task` and `generic.resource`